src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx

```diff
@@ -299,7 +299,13 @@ export const buildStructureLayers = (
 };
 
 /** describes the last arg to buildGsLiteLayers */
+export interface CircleColor {
+  property: string;
+  stops: string[][];
+  type: string;
```

Comment:
how are we using these 2?, I might have missed it.

Reply:
ooh I see, nevermind

Reply:
We are using them for data driven styling.